src/Owin.Security.OpenIdConnect.Server/OpenIdConnectServerHandler.Endpoints.cs

```diff
@@ -1579,12 +1596,18 @@ internal partial class OpenIdConnectServerHandler : AuthenticationHandler<OpenId
 
         private async Task InvokeValidationEndpointAsync() {
             OpenIdConnectMessage request;
-
+            
+            // Note: The updated RFC standard says nothing about allowing GET requests beyond
```

Comment:
If you think removing GET support is better (and you'd probably be right), I'm fine :+1: